sem_covid/services/ml_pipelines/pwdb_classifiers_pipeline.py

```diff
@@ -0,0 +1,244 @@
+#!/usr/bin/python3
+
+# pwdb_classifiers_pipeline.py
+# Date:  01/07/2021
+# Author: Stratulat Ștefan
+
+"""
+    This module aims to define a pipeline for driving classifiers based on the PWDB dataset.
```

Comment:
you mean deriving or training, right? Certainly not a taxi service to drive them around the town :D